pom.xml

```diff
@@ -60,8 +60,8 @@
   <properties>
     <revision>3</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.baseline>2.479</jenkins.baseline>
-    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
+    <jenkins.baseline>2.504</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}.1</jenkins.version>
```

Comment:
This bump is required due to the `workflow-api`  bump.
We can file this bump as a separate PR if requested.

Reply:
Seems it's already been filed: https://github.com/jenkinsci/opentelemetry-plugin/pull/1168